vendor-src/grunt-compile-handlebars/tasks/compile-handlebars.js

```diff
@@ -188,6 +188,7 @@ module.exports = function(grunt) {
 
     handlebarsPath = config.handlebars ? path.resolve(config.handlebars) : 'handlebars';
     handlebars = require(handlebarsPath);
+    handlebars = require(handlebarsPath).create();
```

Comment:
allows us to create isolated instances for when this is executed twice in the same process lifespan